### DISCUSSION TITLE
Is there any alternate for flask

### DISCUSSION THREAD

**Discussion by Vishwaa-Arumugam** [community/community#134783] [Programming Help]
### Body

Is there any alternate for flask other than django and FAstAPI

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by RanjithSrikanth** [comment]
yes django is alternate for flask

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/134782#discussioncomment-10258087) for reference to why this discussion was closed and locked. Thanks!
